src/components/Layout/Footer/Footer.tsx

```diff
@@ -65,6 +65,12 @@ const Footer: FC<FooterProps> = ({ footerLinksSections }: FooterProps) => {
         </nav>
 
         <nav className="mzp-c-footer-secondary">
+          <p>
+            {strings.disclaimer_footer_start}{" "}
+            <a href="https://cloud.google.com/speech-to-text">Google Speech-to-Text</a>{" "}
```

Comment:
Add `target="_blank"` and `rel="noopener noreferrer external"` to `a` so that the external link opens in a new tab (the rels are for security reasons if you are curious).
For example https://github.com/CouncilDataProject/cdp-frontend/blob/6a9b2857bcd414285c4b61ff800f01e7f707fd8e/src/components/Layout/Footer/Footer.tsx#L72